federated_learning/openfl/openfl_mednist_2d_registration/run_docker.md

```diff
@@ -118,7 +118,7 @@ $ fx envoy start --shard-name env_two --disable-tls --envoy-config-path envoy_co
 ```
 [13:48:42] INFO     🧿 Starting the Envoy.                                                                                                            envoy.py:53
 Downloading...
-From: https://drive.google.com/uc?id=1QsnnkvZyJPcbRoV_ArW8SnE1OTuoVbKE
+From: https://developer.download.nvidia.com/assets/Clara/monai/tutorials/MedNIST.tar.gz
```

Comment:
We had stored MedNIST before in Github like [this](https://github.com/Project-MONAI/MONAI/blob/d02ba11d8069870d71316a616f047c499627c71c/monai/apps/datasets.py#L86C65-L86C110). We could use that URL here instead? It seems to still work fine.

Reply:
Yes, update it, thanks!